src/screens/Profile/DesignSystem.tsx

```diff
@@ -133,12 +133,12 @@ export default function DesignSystem() {
 
         <Sections.Section withPadding withTopPadding>
           <Sections.LocationInput
-            label="Label"
+            label="body__tertiary"
```

Comment:
Wrongly replaced text here, should be "Label"? Same with next two?

Reply:
Hah. Fantastic catch! I should have done a case-sensitive replacement AND actually verified. Lazy

Reply:
Fixed